content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -36,84 +49,248 @@ This section of the vehicle form relates to the person who is currently driving
 any plans for a change in the driver in the future, and when.
 
 - :guilabel:`Driver`: select the driver from the drop-down menu, or type in a new driver and click
-  either :guilabel:`Create` or :guilabel:`Create and Edit`.
+  either :guilabel:`Create "driver"` or :guilabel:`Create and edit...` to :ref:`create a new driver
+  and edit the driver details <fleet/new_vehicle/add-driver>`.
+
+  .. important::
+     A driver does **not** have to be an employee. The information for all drivers is stored within
+     the *Fleet* application. If the *Contacts* application is installed, it is also stored there.
+
+     When creating a new driver, the driver is added to both the *Fleet* application and the
+     *Contacts* application, **not** the *Employees* application.
+
+     If the *Contacts* application is not installed when a driver is added, the data is stored
+     within the *Fleet* application. Upon installation of the *Contacts* application, all drivers
+     created in the *Fleet* application are also visible in the *Contacts* application.
```

Comment:
There seems to be a bit of duplicated information here -- would you mind cleaning this up a bit, in order to more succinctly summarize this portion? Thanks!